Libraries/Text/RCTTextAttributes.m

```diff
@@ -17,6 +17,17 @@
 
 @implementation RCTTextAttributes
 
+// [TODO(macOS GH#774)
++ (RCTUIColor *)defaultForegroundColor
+{
+  if (@available(iOS 13.0, *)) {
+    return [RCTUIColor labelColor];
+  } else {
+    return [RCTUIColor blackColor];
```

Comment:
i think macOS 10.10 start having NSColor labelcolor. 10.0 had nscolor textcolor as semantic colors

Reply:
Since we only target >= macOS 10.14 and RCTUIColor is a typedef for NSColor

https://github.com/microsoft/react-native-macos/blob/main/React/Base/RCTUIKit.h#L279

This should be good.

Reply:
10.15 actually. We bumped that starting 0.66 :)